clrdefinitions.cmake

```diff
@@ -189,6 +189,7 @@ if(CLR_CMAKE_PLATFORM_UNIX_AMD64)
 endif (CLR_CMAKE_PLATFORM_UNIX_AMD64)
 add_definitions(-DFEATURE_USE_ASM_GC_WRITE_BARRIERS)
 if(CLR_CMAKE_PLATFORM_ARCH_AMD64 AND NOT WIN32)
+  # add_definitions(-DFEATURE_MANUALLY_MANAGED_CARD_BUNDLES)
```

Comment:
I will un-comment this. I'm making sure that everything is good with and without this feature defined.